tests/get_or_create/tests.py

```diff
@@ -126,6 +126,22 @@ def test_get_or_create_on_related_manager(self):
         # The publisher should have three books.
         self.assertEqual(p.books.count(), 3)
 
+    def test_defaults_exact(self):
+        obj, created = Person.objects.get_or_create(
+            first_name='George', last_name='Harrison', defaults__exact='testing', defaults={
+                'birthday': date(1943, 2, 25), 'defaults': 'testing'
```

Comment:
I'd move `'defaults': 'testing'` to the next line and include a trailing comma per our usual style.